builds/misc/templates/build-packages.yaml

```diff
@@ -0,0 +1,145 @@
+              arch: amd64
+              target.iotedged: edgelet/target/release
+            Debian9-arm32v7:
+              os: debian9
+              arch: arm32v7
+              target.iotedged: edgelet/target/armv7-unknown-linux-gnueabihf/release
+            Debian9-aarch64:
+              os: debian9
+              arch: aarch64
+              target.iotedged: edgelet/target/aarch64-unknown-linux-gnu/release
+  
+            Debian10-amd64:
+              os: debian10
+              arch: amd64
+              target.iotedged: edgelet/target/release
+            Debian10-arm32v7:
+              os: debian10
+              arch: arm32v7
+              target.iotedged: edgelet/target/armv7-unknown-linux-gnueabihf/release
+            Debian10-aarch64:
+              os: debian10
+              arch: aarch64
+              target.iotedged: edgelet/target/aarch64-unknown-linux-gnu/release  
+            Debian11-amd64:
+              os: debian11
+              arch: amd64
+              target.iotedged: edgelet/target/release
+            Debian11-arm32v7:
+              os: debian11
+              arch: arm32v7
+              target.iotedged: edgelet/target/armv7-unknown-linux-gnueabihf/release
+            Debian11-aarch64:
+              os: debian11
+              arch: aarch64
+              target.iotedged: edgelet/target/aarch64-unknown-linux-gnu/release
+              
+            Ubuntu1804-amd64:
+              os: ubuntu18.04
+              arch: amd64
+              target.iotedged: edgelet/target/release
+            Ubuntu1804-arm32v7:
+              os: ubuntu18.04
+              arch: arm32v7
+              target.iotedged: edgelet/target/armv7-unknown-linux-gnueabihf/release
+            Ubuntu1804-aarch64:
+              os: ubuntu18.04
+              arch: aarch64
+              target.iotedged: edgelet/target/aarch64-unknown-linux-gnu/release
+    
+            Ubuntu2004-amd64:
+              arch: amd64
+              os: ubuntu20.04
+              target.iotedged: edgelet/target/release
+            Ubuntu2004-arm32v7:
+              arch: arm32v7
+              os: ubuntu20.04
+              target.iotedged: edgelet/target/armv7-unknown-linux-gnueabihf/release
+            Ubuntu2004-aarch64:
+              arch: aarch64
+              os: ubuntu20.04
+              target.iotedged: edgelet/target/aarch64-unknown-linux-gnu/release
+        steps:
+          - bash: |
+              BASE_VERSION=`cat $BUILD_SOURCESDIRECTORY/edgelet/version.txt`
+              VERSION="$BASE_VERSION$BUILD_BUILDNUMBER"
+              echo "Version: $VERSION"
+              echo "##vso[task.setvariable variable=VERSION;]$VERSION"
+    
+              echo "##vso[task.setvariable variable=PACKAGE_ARCH;]$(arch)"
+              echo "##vso[task.setvariable variable=PACKAGE_OS;]$(os)"
+            displayName: Set Version
+            condition: or(eq(${{ parameters['E2EBuild'] }}, false), eq(variables.arch,'amd64'))
```

Comment:
I added a condition here to make sure that we only build amd64 packages for E2E Build